### PR TITLE
Merge 3.4

### PR DESCRIPTION
**Pull request by alalek**
#602 from l-bat:batchnorm3d
#604 from dkurt:tf_strided_slice
#606 from catree:feat_ippe
#608 from dianlujitao:onnx-log-softmax

Main PR: https://github.com/opencv/opencv/pull/14615
Previous "Merge 3.4": #605

### CODE REVIEW

**Comment by alalek** [comment]
:+1:
